### PR TITLE
valet: add page

### PR DESCRIPTION
**Pull request by stets**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/osx/valet.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# valet
+
+> A minimalist Laravel development environment that allows hosting sites via local tunnels on http://<project>.test
```
Comment (sbrl) [inline pages/osx/valet.md]: Oops! Looks like you're missing a full stop on the end there. What about this?
```suggestion
> A Laravel development environment that allows hosting sites via local tunnels on `http://{{project}}.test`.
```

File: pages/osx/valet.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- View your 'parked' paths:
+
+`valet paths`
+
+- Serve a single site instead of an entire directory:
+
+`valet link app-name`
+
+- Share a project via Ngrok tunnel:
```
Comment (sbrl) [inline pages/osx/valet.md]: Perhaps this will read better?
```suggestion
- Share a project via an Ngrok tunnel:
```

File: pages/osx/valet.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Start the valet daemon:
+
+`valet start`
+
+- Register your current working directory as a path that Valet should search for sites:
+
+`valet park`
+
+- View your 'parked' paths:
```
Comment (andrik) [inline pages/osx/valet.md]: As the term 'parked' is not used in the previous example, I suggest using a similar description that way we could relate the examples
```suggestion
- View your registered paths on Valet:
```

Comment (agnivade) [reply pages/osx/valet.md]: It is indeed mentioned in the previous example. `valet park`

File: pages/osx/valet.md (hunk 1 above)
Comment (schneiderl) [inline pages/osx/valet.md]: For examples urls/hostnames we usually use example.x. Something like 
```suggestion
> A minimalist Laravel development environment that allows hosting sites via local tunnels on http://<example>.test
```
Do you think it makes sense to change it to that?

File: pages/osx/valet.md (hunk 3 above)
Comment (stets) [reply pages/osx/valet.md]: yes!

File: pages/osx/valet.md (hunk 2 above)
Comment (stets) [reply pages/osx/valet.md]: Looks good!

File: pages/osx/valet.md (hunk 1 above)
Comment (stets) [reply pages/osx/valet.md]: done

**Comment by stets** [comment]
> [CLA assistant check](https://cla-assistant.io/tldr-pages/tldr?pullRequest=3266)
> Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/tldr-pages/tldr?pullRequest=3266) before we can accept your contribution.
> **1** out of **2** committers have signed the CLA.
> 
> ✅ stets
> ❌ Stetson Blake
> 
> **Stetson Blake** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tldr-pages/tldr?pullRequest=3266) it.

recommitted. I'm using git on a new macbook so I think that's what the second name is?

**Comment by sbrl** [comment]
Oops! It looks like those commits aren't made by an email address that's added to your GitHub account, @stets. @CLAassistant requires this to verify that you've signed the CLA.

**Comment by stets** [comment]
blew away that last commit -- looking good now

File: pages/osx/valet.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# valet
+
+> A minimalist Laravel development environment that allows hosting sites via local tunnels on `http://<example>.test`.
+> More information: <https://laravel.com/docs/5.8/valet>.
+
+- Start the valet daemon:
+
+`valet start`
+
+- Register your current working directory as a path that Valet should search for sites:
```
Comment (agnivade) [inline pages/osx/valet.md]: Let's change "your" to "the". We prefer to keep the tone formal.

File: pages/osx/valet.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`valet park`
+
+- View your 'parked' paths:
```
Comment (agnivade) [inline pages/osx/valet.md]: We can just remove "your" here. The meaning remains the same.

File: pages/osx/valet.md (hunk 4 above)
Comment (stets) [reply pages/osx/valet.md]: done

File: pages/osx/valet.md (hunk 5 above)
Comment (stets) [reply pages/osx/valet.md]: done

File: pages/osx/valet.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# valet
+
+> A minimalist Laravel development environment that allows hosting sites via local tunnels on `http://<example>.test`.
```
Comment (andrik) [inline pages/osx/valet.md]: Looks like the first suggestion of @sbrl was undo, maybe it was the problem with the commit user

```suggestion
A Laravel development environment that allows hosting sites via local tunnels on `http://<example>.test`.
```

Comment (stets) [reply pages/osx/valet.md]: good catch. fixed

**Pull request merged**